src/makers/controllino/index.ts

```diff
@@ -0,0 +1,9 @@
+import antennas from './antennas'
+import hotspots from './hotspots'
+
+export default {
+  antennas,
+  hotspots,
+  id: 17,
+  supportEmail: 'helium@controllino.com',
```

Comment:
this email is different than the diagnostics opt-out email. intended?

Reply:
Hi @cokes518 yes it is, exactly.